packages/sdk/src/wallet/node/providers/hosted/turnkey/TurnkeyHostedWalletProvider.ts

```diff
@@ -22,15 +24,13 @@ export class TurnkeyHostedWalletProvider extends HostedWalletProvider<
   /**
    * Create a new Turnkey wallet provider
    * @param client - Turnkey client instance (HTTP, server, or core SDK base)
-   * @param organizationId - Turnkey organization ID that owns the signing key
    * @param chainManager - Chain manager used to resolve chains and RPC transports
    */
   constructor(
     private readonly client:
       | TurnkeyHttpClient
       | TurnkeyServerClient
       | TurnkeySDKClientBase,
-    private readonly organizationId: string,
```

Comment:
I realized that `organizationId` should actually be provided at wallet creation and not at verbs creation so removed it from the `WalletProvider` constructor and into the `toVerbsWallet` arguments

Reply:
it's not a static id. depending on how the user has their Turnkey organization set up there might be a different organization id per wallet. Turnkey has a hierarchical organization structure.  More info [here](https://docs.turnkey.com/embedded-wallets/sub-organizations-as-wallets#sub-organizations-as-end-user-controlled-wallets)

Reply:
oh wow okay lets keep this. thanks for explaining!